### DISCUSSION TITLE
Diacritics in pdf output are not displayed properly

### DISCUSSION THREAD

**Discussion by liveweb22** [dompdf/dompdf#3049] [General]
Hello, I also encounter a problem with dompdf, more precisely with the transformation of the text from HTML to PDF. I use the Tiny editor and when I want to preview the text entered with diacritics, the PDF output displays the character: ?
I mention that the database is utf8_ci

https://ibb.co/YhPSKTZ
https://ibb.co/1dDrb57

Any help?

**Comment by bsweeney** [comment]
You'll want to make sure that you're styling your text with a font that supports the characters in the document. The core PDF fonts (Helvetica, Times) have limited character support since they are Windows ANSI. Bundled with Dompdf (up to and including 2.0.1) is the DejaVu fonts (e.g., DejaVu Sans) that has a greater range of supported characters.
